projects/wave-core/src/lib/operators/types/gdal-source-type.model.ts

```diff
@@ -158,6 +173,7 @@ export class GdalSourceType extends OperatorType {
             channelConfig: options && options.channelConfig ? options.channelConfig : this.channelConfig,
             sourcename: this.sourcename,
             transform: this.transform,
+            gdal_params: this.gdalParams ? this.gdalParams : undefined,
```

Comment:
same here

Reply:
variable is not required by mapping here